JavaCalculator/Calculator1.java

```diff
@@ -0,0 +1,58 @@
+import java.util.Scanner;
+
+public class Calculator1 {
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        while (true) {
+            System.out.println("Chose an action");
+            System.out.println("1. Adding (+)");
+            System.out.println("2. Subtraction (-)");
+            System.out.println("3. Multiplying (*)");
+            System.out.println("4. Division (/)");
+            System.out.println("5. Modulus (%)");
+            System.out.print("6. Exit");
+            int person = sc.nextInt();
```

Comment:
it shouldn't be "person"

Reply:
Thank you for the remark.
The mistake is corrected.